### DISCUSSION TITLE
HELP ME

### DISCUSSION THREAD

**Discussion by lhfwhahuwq** [github123666/cidaren#19] [Q&A]
why I put "github123666" into cmd ,it reveals "name 'exit' is not defined"???
i really need this tool to help me complete many test

**Comment by lhfwhahuwq** [comment]
兄弟，怎么使用code啊？
config/config.json这个在哪里？

**Comment by lhfwhahuwq** [comment]
第一步要干什么？？

**Reply by lhfwhahuwq** [reply]
作者不是说在cmd里运行么？
config那个在哪里啊？

**Reply by lhfwhahuwq** [reply]
兄弟我+你vx好不好？帮帮我

**Reply by lihuhuhu** [reply]
exe在releases里面下载v3.7的压缩包，解压。

**Reply by lhfwhahuwq** [reply]
兄弟，加vx

**Reply by lihuhuhu** [reply]
不好意思没时间,文档已经很详细了

**Reply by lhfwhahuwq** [reply]
好的。我电脑没显示那些图片，刚刚用手机看了才显示。我回去再试试。
